packages/syft/src/syft/util/util.py

```diff
@@ -451,11 +450,15 @@ def prompt_warning_message(message: str, confirm: bool = False) -> bool:
     warning = SyftWarning(message=message)
     display(warning)
 
-    if confirm:
-        allowed = Confirm.ask("Would you like to proceed?")
-        if not allowed:
+    while confirm:
```

Comment:
Won't this lock the loop? @jcardonnet

Reply:
@tcp I reproduced the way Rich behaves. It will keep asking until you reply Y or N.